packages/grafana-schema/src/common/mudball.cue

```diff
@@ -190,6 +190,9 @@ BigValueJustifyMode: "auto" | "center" @cuetsy(kind="enum")
 // TODO docs
 BigValueTextMode: "auto" | "value" | "value_and_name" | "name" | "none" @cuetsy(kind="enum",memberNames="Auto|Value|ValueAndName|Name|None")
 
+// TODO docs
+PercentChangeColorMode: "standard" | "inverted" | "same_as_value" @cuetsy(kind="enum",memberNames="Standard|Inverted|SameAsValue")
```

Comment:
should this be defined in the mudball.cue file vs the stat panel's [panelcfg.cue file](https://github.com/grafana/grafana/blob/65793440d3790659708dc8cdf2b28256ae663259/public/app/plugins/panel/stat/panelcfg.cue#L35) alongside the `showPercentChange` option? I think this option only applies to stat vs more broadly to other panels 🤷‍♂️ though it is being used in bigvalue so on second thought this is probably fine lol